giskard/client/giskard_client.py

```diff
@@ -194,6 +194,14 @@ def initialize_kernel(self, project_key: str):
         python_version = f"{sys.version_info[0]}.{sys.version_info[1]}"
         kernel_name = f"{project_key}_kernel"
 
+        analytics.track(
+            "Get kernel",
+            {
+                "kernel_name": anonymize(kernel_name),
```

Comment:
The event name could be "Init kernel from env"/"Automatic kernel init".

We can add anonymous `project_key` here as well.

Reply:
Done! @Inokinoki